### PR TITLE
Update HSEC-2024-0003 with new fix

### PR DESCRIPTION
**Pull request by TristanCacqueray**

### CODE REVIEW

**Review by blackheaven — APPROVED** [review]

File: advisories/hackage/process/HSEC-2024-0003.md [hunk 1]
```diff
@@ -147,6 +150,7 @@ bump.  Because we expect very few (if any) users will be impacted by
 the behavioural change, the GHC team made a pragmatic decision to
 avoid the disruption that a major version bump would cause.
 
+A follow-up fix was released in ***process-1.6.23.0***.
```
Comment (frasertweedale) [inline advisories/hackage/process/HSEC-2024-0003.md:153]: I think we should explain what was deficient in the initial fix.

Comment (TristanCacqueray) [reply advisories/hackage/process/HSEC-2024-0003.md:153]: I added a note.

**Comment by TristanCacqueray** [comment]
I'll keep this as a WIP since we are waiting for the new fix to be merged and released.

File: advisories/hackage/process/HSEC-2024-0003.md (hunk 1 above)
Comment (frasertweedale) [reply advisories/hackage/process/HSEC-2024-0003.md:153]: Could you please push it? :)

Comment (frasertweedale) [reply advisories/hackage/process/HSEC-2024-0003.md]: Thanks, updated commentary LGTM.

**Review by blackheaven — APPROVED** [review]

**Pull request merged**